test-common/uk/gov/hmrc/alcoholdutyaccount/common/TestData.scala

```diff
@@ -224,13 +234,299 @@ trait TestData extends ModelGenerators {
         items = Seq(
           FinancialTransactionItem(
             subItem = "001",
+            dueDate = None,
             amount = 1000.00
           )
         )
       )
     )
   )
 
+  val singleFullyOutstandingReturn: FinancialTransactionDocument =
+    FinancialTransactionDocument(
+      financialTransactions = Seq(
+        FinancialTransaction(
+          sapDocumentNumber = sapDocumentNumberGen.sample.get,
+          periodKey = Some(periodKey),
+          chargeReference = Some(chargeReferenceGen.sample.get),
+          originalAmount = BigDecimal("9000"),
+          mainTransaction = TransactionType.toMainTransactionType(TransactionType.Return),
+          subTransaction = "6132",
+          outstandingAmount = Some(BigDecimal("9000")),
+          items = Seq(
+            FinancialTransactionItem(
+              subItem = "000",
+              dueDate = Some(ReturnPeriod.fromPeriodKeyOrThrow(periodKey).dueDate()),
+              amount = BigDecimal("9000")
+            )
+          )
+        )
+      )
+    )
+
+  val singlePartiallyOutstandingReturn: FinancialTransactionDocument =
+    FinancialTransactionDocument(
+      financialTransactions = Seq(
+        FinancialTransaction(
+          sapDocumentNumber = sapDocumentNumberGen.sample.get,
+          periodKey = Some(periodKey),
+          chargeReference = Some(chargeReferenceGen.sample.get),
+          originalAmount = BigDecimal("9000"),
+          mainTransaction = TransactionType.toMainTransactionType(TransactionType.Return),
+          subTransaction = "6132",
+          outstandingAmount = Some(BigDecimal("5000")),
+          items = Seq(
+            FinancialTransactionItem(
```

Comment:
In open items, there is only one sub item and that has the aoustanding amount on it. Example:

 {
            "chargeType": "Beer 22+ es",
            "mainType": "Alcohol Duty Return",
            "periodKey": "24AD",
            "periodKeyDescription": "April 2024",
            "taxPeriodFrom": "2024-04-01",
            "taxPeriodTo": "2024-04-30",
            "businessPartner": "xxxx",
            "contractAccountCategory": "51",
            "contractAccount": "xxxx",
            "contractObjectType": "ZADP",
            "contractObject": "xxxx",
            "sapDocumentNumber": "wwwww",
            "sapDocumentNumberItem": "0004",
            "chargeReference": "fffffff",
            "mainTransaction": "6074",
            "subTransaction": "6129",
            "originalAmount": 700.0,
            "outstandingAmount": 100.0,
            "clearedAmount": 600.0,
            "accruedInterest": 1.84,
            "items": [
                {
                    "subItem": "000",
                    "dueDate": "2024-05-24",
                    "amount": 100.0
                }
            ]
        },
        
        
       
(Applies to the whole file)

Reply:
True, fixed. Will need fixing in stubs.